### ISSUE TITLE
test AEE on sintel [FlowNetC and FlownetS]

### ISSUE THREAD

**Issue by jeffbaena** [lmb-freiburg/dispnet-flownet-docker#9]
Hi,
 I am trying to replicate your results of FlowNetC and S respectively on Flying Chairs and Sintel clean.

On FlyingChairs validation I obtain values really close to the ones in the paper:
-  2,9 for FlowNetS . (paper state 2,7)
- AEE of 2,4 for FlowNetC (paper state 2,2)

So here it seems correct

On Sintel Clean I obtain lower values of AEE
- 3,16 FlowNetS (paper state 4,5)
- 2,9 FlowNetC (paper state 4,3)
On Sintel Final with FlowNetC I obtain 4,62 while paper state  5,9.

Results of my trained network are really close to the pretrained network provided, so i think it's a testing issue.
The correct results on FlyingChairs, suggest my script to calculate the EPE is correct.

**Are these results reliable?** I get better results on average of 1,5 pixels on sintel clean and final.

Attached there are some plots of my results 
- What i call APE is AEE
- visualization are done just by plotting the absolute value of the flow, without the proper color coding
- i100k and i600k indicate the number of iterations

Thanks in advance for your help,

Stefano

FlowNetS 
flying chairs validation
![eucl_flownets_flying](https://user-images.githubusercontent.com/18122010/45309969-4cb2be00-b525-11e8-9d9a-b13a69df40d8.png)
Sintel clean
![eucl_flownets-sintel](https://user-images.githubusercontent.com/18122010/45309967-4cb2be00-b525-11e8-83e8-bed8a4dc834d.png)

FlowNetS 
flying chairs validation
![eucl_flownetc_flying](https://user-images.githubusercontent.com/18122010/45309968-4cb2be00-b525-11e8-8ff0-725109ad3969.png)
sintel clean
![eucl_flownetc-sintel](https://user-images.githubusercontent.com/18122010/45309966-4c1a2780-b525-11e8-837f-2a9e98e913b7.png)
sintel final
![final_flownetc-sintel](https://user-images.githubusercontent.com/18122010/45309965-4c1a2780-b525-11e8-81b5-d65180216a7b.png)

**Comment by nikolausmayer** [comment]
Hi,
- what's your training data for the different experiments/networks?
- what sort of augmentation did you use?
- the decoder part of the paper networks only goes up to half-resolution. How about yours?
- what numbers do you get when you run your evaluation script on the results of the pretrained nets?

**Comment by jeffbaena** [comment]
Hi Nikolaus and thanks for the prompt answer!

_-what's your training data for the different experiments/networks?_
I train my networks on FlyingChairs using your list for training and evaluation split (640 for test/validation and the rest for training).
I test my network on Sintel training clean and Flying Chairs. Flying chairs gives a correct AEE. (sorry, in the plots i call it APE, but it is AEE), my results on sintel are ~ 1,5 pixel better than the paper.
Results on FlyingChairs are fine

_what sort of augmentation did you use?_
I did not add augmentation, but I have noticed that it is coded on the train.prototxt file of FlowNetS and C.
_the decoder part of the paper networks only goes up to half-resolution. How about yours?_
If I've understood correctly, you are asking for the Network output size, which correspond to the plots height and width and correspond also to the input size. 
e.g. Sintel input= output: 1024 x 436 pixels
_what numbers do you get when you run your evaluation script on the results of the pretrained nets?_
The results with pretrained networks are really close to the ones I get, that is why I think it is not a training issue.
The results are shown on the plots above and referred as "pretrained".

To sum up: Results on FlyingChairs are fine, results on sintel seem better than paper, but this is suspicious to me: I have used the networks provided in dispflownet as they are. 

Thanks for your help, it is really appreciated!

Regards,
Stefano

**Comment by jeffbaena** [comment]
Dear Nikolaus,
I found the mistake, it was a bug on my code and I was not loading the full dataset. 
Sorry for disturbing.

Kind Regards,
Stefano
